src/containers/Main.js

```diff
@@ -30,22 +44,28 @@ const Main = () => {
   return (
     <div className={isDark ? "dark-mode" : null}>
       <StyleProvider value={{isDark: isDark, changeTheme: changeTheme}}>
-        <Header />
-        <Greeting />
-        <Skills />
-        <StackProgress />
-        <Education />
-        <WorkExperience />
-        <Projects />
-        <StartupProject />
-        <Achievement />
-        <Blogs />
-        <Talks />
-        <Twitter />
-        <Podcast />
-        <Profile />
-        <Footer />
-        <Top />
+        {isShowingSplashAnimation && splashScreen.enabled ? (
```

Comment:
We can use `splashScreen.enabled` as a conditional in the useEffect above to completely skip setting the timer if it isn't enabled.